config/samples/dataplane_v1beta1_openstackdataplanenode_edpm_compute_0_full.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNode
+metadata:
+  name: edpm-compute-0
+spec:
+  hostName: edpm-compute-0
+  ansibleHost: 192.168.122.100
+  ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+  node:
```

Comment:
by the way i assume at somepoint we can update that with the NovaExternalComputeTemplate

its not currently in the CRD but we will need that sooner rather then later